actions/transaction_brokernodes.go

```diff
@@ -74,3 +83,47 @@ func (usr *TransactionBrokernodeResource) Create(c buffalo.Context) error {
 
 	return c.Render(200, r.JSON(res))
 }
+
+func (usr *TransactionBrokernodeResource) Update(c buffalo.Context) error {
+	req := transactionUpdateReq{}
+	parseReqBody(c.Request(), &req)
+
+	// Get transaction
+	t := &models.Transaction{}
+	transactionError := models.DB.Eager("DataMap").Find(t, c.Param("id"))
```

Comment:
Is that supposed to say "DataMap" there?

Reply:
NVM, disregard.

Reply:
yeh it's eager loading that association